E-Hospital/E-Hospital.BusinessLogic/Services/Implementation/UserService.cs

```diff
@@ -89,10 +89,10 @@ public void SendVisitRequest(VisitRequestDto visitRequest)
         {
             _visitRequestRepository.Add(_mapper.Map<VisitRequest>(visitRequest));
 
-            var selectedDoctor = _activePatients.FirstOrDefault(x => x.Key.Id == visitRequest.Doctor.Id);
+            var selectedDoctor = _activeDoctors.FirstOrDefault(x => x.Key.Id == visitRequest.Doctor.Id);
             if (selectedDoctor.Key != null)
             {
-                selectedDoctor.Value.UpdateRequestState(visitRequest);
+                selectedDoctor.Value.UpdatePendingRequests(visitRequest);
             }
```

Comment:
There are no sense in `if` statement, because this checking exists in `UpdatePendingRequests` method.